### PR TITLE
Feature/transaction hash

### LINKED ISSUE #319: Support tracking tx that is already signed
Add method to track an already signed transaction given its hash. This message can initiate a `txSent` message although it would not know additional params.

### PR DESCRIPTION
**Pull request by aaronbarnardsound**
@cmeisl Let me know if this implementation which makes use of the `Transaction` API we already have is the best way to go about adding this feature.

Closes #319

### CODE REVIEW

**Comment by aaronbarnardsound** [comment]
Oops I forgot to push the doc updates!

File: README.md [hunk 1]
```diff
@@ -534,22 +534,22 @@ var myContract = assistInstance.Contract(address, abi)
...
 
 #### Returns
 
-`Promise` or `PromiEvent` (`web3.js 1.0`)
+`Promise` or `PromiEvent` (`web3.js 1.0`) if passed a transaction object or `undefined` if passed a transaction hash
 
-- resolves with transaction hash
-- rejects with an error message
+- resolves with transaction hash if passed a transaction object
+- rejects with an error message if pass
```
Comment (cmeisl) [inline README.md]: ?

File: README.md [hunk 2]
```diff
@@ -534,22 +534,22 @@ var myContract = assistInstance.Contract(address, abi)
...
+`txObjectOrHash` - `Object` || `String`: Transaction object or transaction hash (**Required**)
 
 `callback` - `Function`: Optional error first style callback if you don't want to use promises
 
 `inlineCustomMsgs` - `Object`: Optional notification message overrides
 
 #### Returns
 
-`Promise` or `PromiEvent` (`web3.js 1.0`)
+`Promise` or `PromiEvent` (`web3.js 1.0`) if passed a transaction object or `undefined` if passed a transaction hash
```
Comment (cmeisl) [inline README.md]: we return 'undefined' if passed a tx hash? I understand there is no promise to return, but perhaps we should return something like `true` if the passed hash is valid and valid if it's not?

Comment (aaronbarnardsound) [reply README.md]: Agreed. Updated.

File: README.md (hunk 1 above)
Comment (aaronbarnardsound) [reply README.md]: Fixed

**Pull request merged**